Mod Menu Helper/assets/modmenu/lang/tr_TR.json

```diff
@@ -0,0 +1,55 @@
+  "modmenu.summaryTranslation.betterbeds": "§c*§r Hızlı yataklar\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.bettermounthud": "Atın üstündeyken yemek ve XP'yi göster\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.bobby": "Sunuculardaki yüklü yığınları yüklü tutar\n§9✎§8 Varlıkları blokla, yığınları azalt, önbelleği koru, ...",
+  "modmenu.summaryTranslation.borderlessmining": "Tam ekrandayken Alt+Tab desteği (opt-in)\n§9✎§8 Modu, macOS desteğini, özel boyut vb. aç/kapa",
+  "modmenu.summaryTranslation.capes": "Optifine veya diğerlerinden pelerinler\n§9✎§8 Türler, tercih edilen tür, elitra, ...",
+  "modmenu.summaryTranslation.citresewn": "Optifine'ın özel eşyaları\n§9✎§8 Modu aç/kapa, bozuk paketlere izin ver, ...",
+  "modmenu.summaryTranslation.colormatic": "Optifine'ın özel renkleri\n§9✎§8 Gökyüzünü veya yer boşluğunu temizle, ışık yumuşatma",
+  "modmenu.summaryTranslation.continuity": "Optifine'ın bağlantılı dokuları\n§9✎§8 Modu, metodu, kaynak paketlerini aç/kapa",
+  "modmenu.summaryTranslation.cullleaves": "Optifine'ın \"daha iyi ağaçları\"\n§9✎§8 Modu, kaynak paketini aç/kapa",
+  "modmenu.summaryTranslation.cem": "Optifine'ın özel varlıkları (alfa)\n§9✎§8 Optifine desteği, model düzeltimi",
+  "modmenu.summaryTranslation.dcch": "Farklı dünyalardan yazım geçmişini tut\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.dynamicfps": "Arkaplandayken FPS'i düşür\n§9✎§8 Modu aç/kapa, üzerindeyken normale çevir, ...",
+  "modmenu.summaryTranslation.enhancedblockentities": "Daha hızlı sandıklar, tabelalar, yataklar vb.\n§9✎§8 Geliştirilecek bloklar",
+  "modmenu.summaryTranslation.entityculling": "Engellerin arkasındaki mobları görüntüleme\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.entity_texture_features": "Optifine'daki rastgele ve ışık emen moblar\n§9✎§8 Rastgele moblar, ışık emiciliği, göz kırpabilmeleri, ...",
+  "modmenu.summaryTranslation.fabrishot": "F9 ile yüksek çözünürlüklü görüntü al\n§9✎§8 F2 tuşunu, çözünürlüğü, tuş ataması vb. üzerine yaz.",
+  "modmenu.summaryTranslation.farsight": "§c*§r Sunuculardaki yüklü yığınları yüklü tutar\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.fastchest": "§c*§r Daha hızlı sandıklar\n§9✎§8 Modu aç/kapa",
+  "modmenu.summaryTranslation.fastopenlinksandfolders": "Bağlantı veya klasörleri açarken donmayı önler\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.ferritecore": "RAM kullanımı azaltır\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.forgetmechunk": "DFixes lag spikes on chunk borders\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.hydrogen": "Reduce RAM usage\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.iris": "Optifine'ın shaderları\n§9✎§8 Modu aç/kapa, shader, tuş atamaları vb. seç",
+  "modmenu.summaryTranslation.itemmodelfix": "Elde tutulan eşyadaki görünmüz çizgileri düzeltir\n§9✎§8 Metod",
```

Comment:
Context: just look at the comparison images on Curseforge: https://www.curseforge.com/minecraft/mc-mods/item-model-fix

Reply:
I understood the description itself, but didn't get the things after line break which directly related to the context on line 2 & 3 you provided. Translation should be OK as it is.